Bug 1174367 - remove the #xywh media fragment in crop_resize_rotate.js

davidflanagan opened the pull request.

Commit:
Bug 1174367 - remove the #xywh media fragment in crop_resize_rotate.js

lightsofapollo commented:
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7e3a91454daed3df7564385e02de6856c0d3ebfe

punamdahiya commented on shared/js/media/crop_resize_rotate.js:
```diff
@@ -351,12 +347,7 @@ function cropResizeRotate(blob, cropRegion, outputSize, outputType,
     // that we do not want it to decode all of the pixels in the image. The
     // #-moz-samplesize= fragment allows us to specify that JPEG images
     // should be downsampled while being decoded, and this can save a lot of
-    // memory. If we are not going to downsample the image, but are going to
```
Comments in Line 199 and Line 212 still mention #xywh media fragment, now that we are removing #xywh media fragment, we can delete those comments.

autolander commented:
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=27c8142d3b0273b0a608b2aeb1e644b42ba4194f

davidflanagan commented:
Addressed review comments. The change only modifies comments, and the tests were green before, so I'm going to land without waiting for this new test run.
